packages/frontend-2/components/viewer/comments/Comments.vue

```diff
@@ -118,6 +119,11 @@ graphql(`
   }
 `)
 
+const discussionLoadedVersionOnly = useSynchronizedCookie<boolean>(
```

Comment:
Would be cleaner to put all of this together with where threadFilters actually get set up. In setup.ts `setupResourceRequest()` we define the empty `threadFilters` ref.